### PR TITLE
feat: Configuration on account page

### PR DESCRIPTION
**Pull request by EdenComp**
# Description

This PR adds a second card under the Account card, the Configuration page.
It replaces the old component ConfigPage

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [X] I have added the needed labels
- [X] I have linked this PR to an issue
- [X] I have linked this PR to a milestone
- [X] I have linked this PR to a project
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

There is currently some problems with the theme: will investigate

### CODE REVIEW

File: src/components/ConfigPage.tsx [hunk 1]
```diff
@@ -41,7 +54,217 @@ const ConfigPage = (): JSX.Element => {
...
+									bgGradient={`linear-gradient(90deg, ${colors.blue[900]} 0%, ${colors.blue[900]} 100%)`}
+									fontWeight="700"
+								>
+									[userName]@[repositoryName]
+								</Box>
+							</Text>
+						</Stack>
+					</Stack>
+				</VStack>
+			</HStack>
```
Comment (RezaRahemtola) [inline src/components/ConfigPage.tsx:265]: Try to split this in multiple components, it's pretty huge right now :smile:

**Review by RezaRahemtola — CHANGES_REQUESTED** [review]
The global design is here but the values are missing.

Don't forget to handle responsive and keep future configuration options in mind :+1:

**Review by RezaRahemtola — CHANGES_REQUESTED** [review]
When testing locally I can't edit the configuration options (except for the light/dark theme), which was possible before.

**Review by RezaRahemtola — APPROVED** [review]

**Review by lucas-louis — APPROVED** [review]
LGTM

**Review by RezaRahemtola — APPROVED** [review]

**Pull request merged**